dashboard/charts.py

```diff
@@ -85,19 +85,18 @@ def create_choropleth_map(locations):
 
 
 @st.cache_data
-def get_albums_sales_line_graph(chosen_album) -> alt.Chart:
+def get_artist_sales_line_graph(chosen_artist) -> alt.Chart:
     """Returns a line graph of sales over time for a chosen album."""
-    return alt.Chart(chosen_album).mark_line().encode(
+    return alt.Chart(chosen_artist).mark_line().encode(
         x=alt.X("timestamp:T"),
         y=alt.Y("sales:Q"),
-        color="name:N"
     ).interactive()
 
 
 @st.cache_data
-def get_tag_sales_line_graph(chosen_tag) -> alt.Chart:
+def get_sales_line_graph(chosen_data) -> alt.Chart:
     """Returns a line graph of sales over time for a chosen tag"""
```

Comment:
Does this docstring also need to be changed?

Reply:
Yes just changed it